CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ API Changes:
 - Enable `CaptureFailedRequests` by default ([2688](https://github.com/getsentry/sentry-dotnet/pull/2688))
 - Additional constructors removed from `TransactionTracer`. ([#2694](https://github.com/getsentry/sentry-dotnet/pull/2694))
 - Removed the `Scope.Platform` property as it was never applied ([#2695](https://github.com/getsentry/sentry-dotnet/pull/2695))
+- `WithScope` and `WithScopeAsync` have been proven to not work correctly in desktop contexts when using a global scope. They are have been removed in favor of the overloads of `CaptureEvent`, `CaptureMessage`, and `CaptureException`. Those methods provide a callback to a configurable scope. ([#2717](https://github.com/getsentry/sentry-dotnet/pull/2717))
```

Comment:
```suggestion
- `WithScope` and `WithScopeAsync` have been proven to not work correctly in desktop contexts when using a global scope. Removing them in favor of the overloads of `CaptureEvent`, `CaptureMessage`, and `CaptureException`. Those methods provide a callback to a configurable scope that doesn't . ([#2717](https://github.com/getsentry/sentry-dotnet/pull/2717))
```

I think it would be nice to expand. Like: Before, After

WIthScope(..); // Becomes:

CaptureEvent(..).

Reply:
> That doesn't

That doesn't what? That doesn't not work correctly? If so, I'd take out the double negative and go with "that work correctly when using a global scope" or something like that.